enhancements/.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- 
+	Please fill out the following questions when adding/updating an enhancement in the openshift/enhancements repo
+-->
+### Enhancement Description
+
+<!-- short description of enhancement-->  
+- Brief (1-2 lines) enhancement description:
+
+<!-- link to related discussions about this enhancement, for example a github issue, RFE or public mailing list link if available -->
+- Discussion links:
+
+<!-- add which team will be primarily responsible for owning and maintaining this enhancement -->
+- Primary team:
+
+<!-- add any other teams that must review or will be impacted by this enhancement -->
```

Comment:
This is definitely redundant. Is the point to list people here so they are pinged?

Reply:
OK, what if we say that we want the approvers and initial reviewers listed here, then?